tests/queries/0_stateless/02525_different_engines_in_temporary_tables.sql

```diff
@@ -0,0 +1,45 @@
+DROP TEMPORARY TABLE IF EXISTS table_merge_tree_02525;
+CREATE TEMPORARY TABLE table_merge_tree_02525
```

Comment:
It would be great to add some negative tests (like trying to create a temporary Replicated table) and tests with multiple sessions

Reply:
I added negative tests for ReplicatedMergeTree and KeeperMap. 
Please advice about "multiple sessions". We cannot test multiple sessions in functional tests (Is it true?). So we have one choice - use integration tests. Is it right? 
What do you think we need to test for multiple sessions?

Reply:
For multiple sessions I saw examples in SH test files. They use curl. I will one test for sessions tomorrow.

Reply:
We can use multiple sessions in `*.sh` functional tests, see, for example, `01601_temporary_table_session_scope` and `01167_isolation_hermitage`. It makes sense to check that two temporary tables with the same name can be used concurrently in different sessions without conflicts (e.g. they will not write to the same directory). Also it makes sense to create a temporary table, but do not drop it in order to check that table is correctly removed automatically in background on session close (but it can be added in the end of the `.sql` test)

Reply:
Based on your suggestions I implemented new test for sessions:
tests/queries/0_stateless/02561_temporary_table_sessions.sh
Please take a look.